src/interfaces/commands/command.ts

```diff
@@ -0,0 +1,5 @@
+import { ICommand } from './command.interface';
+
+export class Command<T> implements ICommand {
+  resultType$e1ca39fa!: T;
```

Comment:
Shouldn't it be `resultType: T;` ?
```suggestion
  resultType: T;
```